Source/Blazorise/Base/BaseRowableComponent.cs

```diff
@@ -9,58 +9,14 @@ namespace Blazorise;
 /// <summary>
 /// Base class for components that are containers for other components.
 /// </summary>
-public abstract class BaseRowableComponent : BaseComponent, IRowableComponent
+public abstract class BaseRowableComponent : BaseComponent
```

Comment:
Maybe BaseRowableComponent was not needed now. The RowContext was added so that there was less spaghetti code with all the `NotifyColumnableInitialized`, `NotifyColumnableRemoved` that was the same across several components. This way the logic is centralized into RowContext.

Reply:
I see. Great.